packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -400,9 +396,9 @@ class OverflowMenu extends Component {
           return nextIndex;
       }
     })();
-    const { overflowMenuItem } =
+    const overflowMenuItem =
       this[`overflowMenuItem${enabledIndices[nextValidIndex]}`];
-    overflowMenuItem?.current?.focus();
+    overflowMenuItem?.focus();
```

Comment:
Is this the root of the bug? There was already code attempting to focus the item but it just wasn't calling `focus()` on the right thing?

Reply:
yeah, there's no "overflowItem" property to destructure from the result of 400 and even then the ".current" was always coming up undefined, I tried just focusing the element directly and that worked. The value is being set here for context:
<img width="663" alt="image" src="https://user-images.githubusercontent.com/40550942/214402808-aa58d5cc-8197-4c3b-a86d-bf54866bf2b8.png">

my theory is maybe this code used to work way back when but then as the component kept getting changed/refactored this broke ?